src/main/java/com/example/onlinestore/model/Product.java

```diff
@@ -0,0 +1,24 @@
+package com.example.onlinestore.model;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.awt.*;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@Entity
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int Id;
```

Comment:
[nitpick] instance field names should start with lowercase:
```suggestion
    private int id;
```

See [naming convention guide](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) (: